src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "pch.h"
 #include "Profile.h"
-#include "Utils.h"
+#include "../inc/Utils.h"

```

Comment:
sorry, _what_ lives in inc/utils.h now? i couldn't tell by searching the PR :) and I can;t find the file in your branch

how does this compile o_O

Reply:
Huh. This was pointing to "WinRTUtils/inc/Utils". Not needed anymore.